### ISSUE TITLE
Bibliography anchors cause parser error if alone on a line

### ISSUE THREAD

**Issue by reitzig** [asciidoctor/asciidoctor-intellij-plugin#614]
## Observed vs. expected behavior

Some bibliography anchors trip up the parser:

![image](https://user-images.githubusercontent.com/4246780/102512138-f466b100-4089-11eb-9489-b00375761abb.png)

`asciidoctor-pdf` accepts the input without warnings, the final document looks as expected and all links resolve.

Side note: The preview doesn't work well at all, here. Not seeing this in my real document, but maybe worth checking out. :thinking: There, the preview shows even those references correctly that are shown as broken in the editor pane!

![image](https://user-images.githubusercontent.com/4246780/102512425-4f98a380-408a-11eb-88d4-700c766bb093.png)

## Steps to reproduce

```adoc
Small text with <<foo>> and <<bar>>.

- [[[foo,1]]]
  On the next line
- [[[bar,2]]] On the same line
```

## Environment

Plugin Version: 0.32.2

IntelliJ Details: IntelliJ IDEA 2020.3 (Ultimate Edition)
Build #IU-203.5981.155, built on November 30, 2020
Runtime version: 11.0.9+11-b1145.21 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.4.0-52-generic
GC: ParNew, ConcurrentMarkSweep
Memory: 1981M
Cores: 8
Registry: debugger.click.disable.breakpoints=true
Non-Bundled Plugins: Key Promoter X, com.intellij.ideolog, google-java-format, mobi.hsz.idea.gitignore, name.kropp.intellij.makefile, org.jetbrains.plugins.go-template, org.nik.presentation-assistant, org.toml.lang, com.intellij.kubernetes, net.ashald.envfile, org.jetbrains.kotlin, com.jetbrains.edu, org.asciidoctor.intellij.asciidoc
Current Desktop: i3

**Comment by ahus1** [comment]
Sorry this is bothering you, and thank you for reporting this. 

Regarding bibliography list items: Having the text on the next line will be fixed in the next release. I'll update this ticket once a preview is available.

Regarding the preview: it will render bib references correctly [only inside a bibliography section, see the user guide for this](https://asciidoctor.org/docs/user-manual/#user-biblio). 

The following snippet

```
[bibliography]
== References

Small text with <<foo>> and <<bar>>.

- [[[foo,1]]]
  test
- [[[bar,2]]] On the same line
```

renders as follows in the preview:

![image](https://user-images.githubusercontent.com/3957921/102539003-5edd1880-40ad-11eb-8c7a-1736af3f9452.png)

**Comment by ahus1** [comment]
This change is now available as part or pre-release 0.32.4.

The pre-release of the plugin is available from [GitHub releases](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html).
